examples/basic_usage/usage.sh

```diff
@@ -8,9 +8,10 @@ metatensor-models train options.yaml
 metatensor-models train --help
 
 # We now evaluate the model on the training dataset, where the first arguments specifies
-# the model and the second the structure file
+# an options file, the second trained model. The ``--eval_on`` indicates that we want to
+# evaluate on the ``training_set`` as defined in the ``options.yaml``.
 
-metatensor-models eval model.pt qm9_reduced_100.xyz
+metatensor-models eval options.yaml model.pt --eval_on="training"
```

Comment:
Not sure about this... should we just allow the user to input a single dataset (or multiple) without specific names? In particular, having a training set makes little sense when you want to evaluate a model. This will be very confusing

Reply:
I did another comment. We can change this but this means people have to write another yaml file.